### ISSUE TITLE
Now Build / npm run build during deployment with Now breaks with React v16.0.0-beta.5

### ISSUE THREAD

**Issue by hamlim** [zeit/next.js#2737]
Attempted to deploy a Next.js project via `now` using React v16.0.0-beta.5, the build fails at:

`now` fails silently, attempted with `now --force`:
```Javascript
{ Error: Cannot find module 'react/dist/react.min.js'
>     at Function.Module._resolveFilename (module.js:485:15)
>     at Function.resolve (internal/module.js:18:19)
>     at _callee2$ (/home/nowuser/src/node_modules/6337955207656a4cd3a86bfedc881b16a7d3fec2/dist/server/build/webpack.js:516:34)
>     at tryCatch (/home/nowuser/src/node_modules/ed1eef08d53a7c75df4193ad02d65438b9ab3abb/runtime.js:65:40)
>     at Generator.invoke [as _invoke] (/home/nowuser/src/node_modules/ed1eef08d53a7c75df4193ad02d65438b9ab3abb/runtime.js:303:22)
>     at Generator.prototype.(anonymous function) [as next] (/home/nowuser/src/node_modules/ed1eef08d53a7c75df4193ad02d65438b9ab3abb/runtime.js:117:21)
>     at step (/home/nowuser/src/node_modules/44690af3a28dc6904408135dcac31bde56bd9346/helpers/asyncToGenerator.js:17:30)
>     at /home/nowuser/src/node_modules/44690af3a28dc6904408135dcac31bde56bd9346/helpers/asyncToGenerator.js:35:14
>     at Promise (<anonymous>)
>     at F (/home/nowuser/src/node_modules/4eb0ac885dfe4f84379aa081d2fd358a78bc9753/library/modules/_export.js:35:28) code: 'MODULE_NOT_FOUND' }
> npm ERR! code ELIFECYCLE
> npm ERR! errno 1
> npm ERR! @ build: `next build`
> npm ERR! Exit status 1
> npm ERR!
> npm ERR! Failed at the @ build script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

I changed React back down to 15.6 and it built successfully.

**Comment by Florian-R** [comment]
See #1098 and #2659

TL;DR; Next don't work (yet) with React 16 but it's in the pipe.
